src/Donations/Micros/PaymentStatus/TransferDetails.tsx

```diff
@@ -44,9 +46,24 @@ function TransferDetails({
       <div className={"title-text text-center"} data-test-id="test-thankYou">
         {t("common:transferDetails")}
       </div>
-      <div className={"mt-20 text-center"}>
-        {t("common:transferDetailsMessage")}
-      </div>
+
+      {frequency === "once" ? (
+        <div className={"mt-20 text-center"}>
+          {t("common:transferDetailsMessage")}
+        </div>
+      ) : (
+        <div className={"mt-20 text-center"}>
+          {t("common:recurrentTransferDetailsMsg", {
+            frequency,
+            amount: getFormatedCurrency(
+              i18n.language,
+              donation.currency,
+              donation.amount
+            ),
+          })}
+        </div>
```

Comment:
This could be refactored slightly, to move the check for `frequency === "once" ` inside one common `<div>` (as both divs have styles `mt-20` and 
`text-center` applied)